ext/tidy/tidy.c

```diff
@@ -269,6 +260,16 @@ static void TIDY_CALL php_tidy_panic(ctmbstr msg)
 	php_error_docref(NULL, E_ERROR, "Could not allocate memory for tidy! (Reason: %s)", (char *)msg);
 }
 
+static void php_tidy_load_config(TidyDoc doc, const char *path)
+{
+	int ret = tidyLoadConfig(doc, path);
+	if (ret < 0) {
+		php_error_docref(NULL, E_WARNING, "Could not load configuration file \"%s\"", path);
+	} else if (ret > 0) {
+		php_error_docref(NULL, E_NOTICE, "There were errors while parsing the configuration file \"%s\"", path);
```

Comment:
Might make sense to improve the error message to mention ``Tidy`` somewhere?

Reply:
That makes sense, hang on I'll quickly do that.